addons/web/static/src/views/fields/boolean_toggle/boolean_toggle_field.js

```diff
@@ -3,25 +3,16 @@
 import { CheckBox } from "@web/core/checkbox/checkbox";
```

Comment:
We can remove the import as it is no more used.

Reply:
Oups indeed. 🙂